src/index.js

```diff
@@ -28,7 +28,13 @@ let stop
 let getEntries
 let clear
 
-if (perf && perf.mark) {
+if (
+  perf &&
+  perf.mark &&
+  perf.getEntriesByName &&
```

Comment:
It's probably enough to stop after getEntriesByName. See https://developer.mozilla.org/en-US/docs/Web/API/Performance vs https://nodejs.org/api/perf_hooks.html

Another solution is to higher up and not try to bind to the performance global if the environment is Node.

Reply:
> It's probably enough to stop after getEntriesByName. See https://developer.mozilla.org/en-US/docs/Web/API/Performance vs https://nodejs.org/api/perf_hooks.html

It's enough *right now* ;)

> Another solution is to higher up and not try to bind to the performance global if the environment is Node.

Sure, I don't have a preference for how @nolanlawson would like to have it fixed so long as it is fixed :)

Reply:
> It's enough right now ;)

It's a stable API on both sides, so there's no good reason it would change. But I can't say that checking for every method is wrong, either ;)

Reply:
I'm fine with checking for every single method. This ensure that some version of Node in the future or some other JS environment doesn't have a similar error. :)